components/predict-form/question-types/MultiChoiceQuestion.tsx

```diff
@@ -60,24 +77,34 @@ export default function MultiChoiceQuestion({
           />
         ))}
 
-        {fields.length < MAX_OPTIONS && (
-          <div className="flex flex-row justify-between items-center gap-2">
-            <div className="flex-grow">
+        <div className="flex flex-row justify-between items-center gap-2">
+          <div className="flex-grow flex gap-2">
+            <button
+              onClick={(e) => {
+                e.preventDefault()
+                addOption()
+              }}
+              className="btn btn-sm text-neutral-500"
+            >
+              <PlusIcon className="w-5 h-5" />
+              Add option
+            </button>
+
+            {
               <button
```

Comment:
Github has diffed this very strangely but if you take a look at the code I'm just adding a new button here along with some styling